### PR TITLE
Add new zip for example SAML configuration files that reflect latest version

### LINKED ISSUE #618: [BUG] Security saml-example is using docker-compose file pointed to `1.0.1`
### What is the bug?
When demonstrating for users how to setup saml for their clusters an example docker compose file is offered to streamline configuration.  This file contains hardcoded references to the `1.0.1` release version of OpenSearch, which is not supported, instead this sample file should use the latest version(s) of OpenSearch/Dashboards.

### How can one reproduce the bug?
See https://github.com/opensearch-project/documentation-website/blob/main/assets/examples/saml-example.zip its hardcoded to a specific version and hasn't been updated since it was added.

### What is the expected behavior?
As newer versions of OpenSearch become available the sample file should be updated.  Note, a `latest` docker version identifier could be used.

Note; this file should also be stored as plaintext instead of a binary and dynamically assembled on website push - feel free to make another issue.

### Do you have any additional context?
From the `docker-compose.yml` file within that zip:
```
version: '3'
services:
  opensearch-saml-node1:
    image: opensearchproject/opensearch:1.0.1
    container_name: opensearch-saml-node1
    environment:
      - cluster.name=opensearch-saml-cluster
      - node.name=opensearch-saml-node1
      - discovery.seed_hosts=opensearch-saml-node1,opensearch-saml-node2
      - cluster.initial_master_nodes=opensearch-saml-node1,opensearch-saml-node2
      - bootstrap.memory_lock=true # along with the memlock settings below, disables swapping
      - "OPENSEARCH_JAVA_OPTS=-Xms512m -Xmx512m" # minimum and maximum Java heap size, recommend setting both to 50% of system RAM
```

### PR DESCRIPTION
**Pull request by cwillum**
### Description
These changes add a new saml-example-custom.zip file containing `customize-docker-compose.yml`, `customize-config.yml`, and `customize-opensearch-dashboards.yml` files for use as the fully functioning SAML example. These changes also include edits to documentation so that steps accord with the new files.

The `opensearch-dashboards.yml` file contains the `server.xsrf.allowlist` setting. For version 1.3 this setting must remain `server.xsrf.whitelist` to conform to code for that version. Therefore, I'll create a new PR for that download. This PR covers versions 2.x and should work for subsequent major releases.

### Issues Resolved
Fixes #618 

### Checklist
- [ ] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by cwillum** [comment]
Since we can't view the contents of the zip file here in GitHub, I'm pasting the note I added to the image settings in the `docker-compose.yml` file directing users to change the version number. The steps also include instruction to perform this edit. But I'm pasting it here for visibility:

`image: opensearchproject/opensearch:latest # replace "latest" with your preferred OpenSearch version number before launching: for example, 2.6.0.`

**Comment by cwillum** [comment]
@opensearch-project/security I'm trying to resolve this issue with the SAML example zip file getting stale with each new version release. I've added steps to have users modify the `docker-compose.yml` file after download with OpenSearch version number in the proper `image` fields. I also added comments in the YAML file instructing users to do the same. The comment in the `docker-compose.yml` file looks like this:
```yml
image: opensearchproject/opensearch:latest # replace "latest" with your preferred OpenSearch version number before launching in production: for example, 2.6.0.
```
Could you have a look at this edit and see if it looks and sounds accurate/makes sense. Thanks.

**Review by peternied — COMMENTED** [review]
I wouldn't recommend the tag `latest`, instead lets use `2`.  This will point to whatever the latest v2.X.X version of OpenSearch that has been released, this should ensure this client will always startup with modifying these instructions.

File: _security/authentication-backends/saml.md [hunk 1]
```diff
@@ -19,13 +19,31 @@ This profile is meant for use with web browsers. It is not a general-purpose way
...
+   * `customize-config.yml`: Configures SAML for authentication.
+
+   You can remove "customize" from the file names if you plan to modify and keep these files for production.
+   {: .tip }  
+
+1. In the `docker-compose.yml` file, specify your OpenSearch version number in the `image` field for nodes 1 and 2, and OpenSearch Dashboards server. For example, if you are running OpenSearch version 2.6, the `image` fields will resemble the following examples:
+   
+   ```yml
+   opensearch-saml-node1:
+    image: opensearchproject/opensearch:2.6.0
```
Comment (Naarcha-AWS) [inline _security/authentication-backends/saml.md:37]:
```suggestion
    image: opensearchproject/opensearch:2.9.0
```

File: _security/authentication-backends/saml.md [hunk 2]
```diff
@@ -19,13 +19,31 @@ This profile is meant for use with web browsers. It is not a general-purpose way
...
+    ```
+    ```yml
+    opensearch-saml-node2:
+    image: opensearchproject/opensearch:2.6.0
```
Comment (Naarcha-AWS) [inline _security/authentication-backends/saml.md:41]:
```suggestion
    image: opensearchproject/opensearch:2.9.0
```

File: _security/authentication-backends/saml.md [hunk 3]
```diff
@@ -19,13 +19,31 @@ This profile is meant for use with web browsers. It is not a general-purpose way
...
+    ```
+    ```yml
+    opensearch-saml-dashboards:
+    image: opensearchproject/opensearch-dashboards:2.6.0
```
Comment (Naarcha-AWS) [inline _security/authentication-backends/saml.md:45]:
```suggestion
    image: opensearchproject/opensearch-dashboards:2.9.0
```

**Review by Naarcha-AWS — APPROVED** [review]

**Pull request merged**